### PR TITLE
UI-III-Flexbox | Megan White

### PR DESCRIPTION
**Pull request by meganwhite**
alasalle

### CODE REVIEW

File: great-idea/index.html [hunk 1]
```diff
@@ -17,7 +17,63 @@
   <body>
```
Comment (alasalle) [inline great-idea/index.html:17]: The title (inside the head part of your index.html page) should be Great Idea! - Services

File: great-idea/index.html [hunk 2]
```diff
@@ -17,7 +17,63 @@
...
+    
+    <div class="main-heading">
+      <div class="heading-text">
+        <h1>Innovation<br>On<br>Demand</h1>
+        <button>Get Started</button>
+      </div>
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+
+    <div class="top-row">
```
Comment (alasalle) [inline great-idea/index.html:40]: Just a tip for the future: some of these divs could work well as sections.

File: great-idea/services.html [hunk 3]
```diff
@@ -16,91 +16,91 @@
...
-Copyright Great Idea! 2018
+  <header>
+    <nav>
+      <a href = 'services.html'>Services</a>
+      <a href = '#'>Product</a>
+      <a href = '#'>Vision</a>
+      <a href = '#'>Features</a>
+      <a href = '#'>About</a>
+      <a href = '#'>Contact</a>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```
Comment (alasalle) [inline great-idea/services.html]: Logo img should be wrapped in an anchor and point to your index.html page.

**Comment by alasalle** [comment]
Overall, a really solid job! Loved the clear class names. Also, a good pull request name format to use: Project Name | Your Name

**Comment by meganwhite** [comment]
alasalle - I made the updates you suggested. Thanks for your comments!

**Pull request merged**